### ISSUE TITLE
init Number is not right

### ISSUE THREAD

**Issue by lqzhgood** [fitnr/sqlite-json#13]
in sqlite 3

```
// table : test
// uniseq: integer

INSERT INTO "main"."test"("uniseq") VALUES (6500328718134052119);
```

```
 const exporter = sqliteJson(path.join(__dirname, '../../11.db'));
    exporter.json(`select * FROM test`, (err, json) => {
        console.log('json', json);
    });
// json [{"uniseq":6500328718134052000}]

```

value is  `6500328718134052119` but got `6500328718134052000`

**Comment by lqzhgood** [comment]
show cover by `BigInt` from db, and 
```
let n = BigInt(x);
n = n > Number.MIN_SAFE_INTEGER && n < Number.MAX_SAFE_INTEGER ? Number(n) : n.toString()
```

**Comment by lqzhgood** [comment]
BigInt.prototype.toJSON = function () {
    return this > Number.MIN_SAFE_INTEGER && this < Number.MAX_SAFE_INTEGER ? Number(this) : this.toString();
};

**Comment by fitnr** [comment]
This was an issue with the underlying `node-sqlite3` package. Fixed by moving to `better-sqlite3` and using a BigInt-aware JSON serializer. Above commit published as v1.2
